plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/BlankDeployPreferencesPanel.java

```diff
@@ -0,0 +1,30 @@
+package com.google.cloud.tools.eclipse.appengine.deploy.ui;
+
+import org.eclipse.core.databinding.DataBindingContext;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+
+public class BlankDeployPreferencesPanel extends DeployPreferencesPanel {
+
+  BlankDeployPreferencesPanel(Composite parent) {
+    super(parent, SWT.NONE);
+    Label noGAEProjectLabel = new Label(this, SWT.NONE);
+    noGAEProjectLabel.setText("Not an App Engine Application");
+  }
+
+  @Override
+  DataBindingContext getDataBindingContext() {
+    return new DataBindingContext();
+  }
+
+  @Override
+  void resetToDefaults() {
+  }
+
+  @Override
+  boolean savePreferences() {
+    return true;
```

Comment:
We use the return value of the method to enable or prevent the OK button's click action to be continued. If we return false here, the users won't be able to close the dialog is this page is open. The deployment preferences manipulated by the other panels won't be affected.